compiler/src/skipError.sk

```diff
@@ -91,8 +91,12 @@ fun throwFlattenedErrors(errors: Vector<Vector<Error>>): void {
 
 fun printErrorsAndExit<T>(
   errors: Array<SkipErrorException>,
-  get_file: String -> (String, String) = filename ->
-    (filename, FileSystem.readTextFile(filename)),
+  get_file: String -> (String, String) = filename -> {
+    if (filename.contains(":")) {
+      !filename = filename.splitFirst(":")[1]
```

Comment:
should be `.i1` rather than `[1]` I think

Reply:
My bad, it's up to date now.